troposphere/static/js/components/images/detail/header/HeaderView.react.js

```diff
@@ -1,14 +1,15 @@
 
 define(
   [
-    'react',
+    'react/addons',
     'backbone',
     'jquery',
     'stores',
+    'modals',
     // plugins
-    'bootstrap'
+    // 'bootstrap' // Not used?
```

Comment:
Not sure about here but we do use some of the other Bootstrap JS components like tool tips and nav menus as well.